lambda_functions/stac/stac_check.py

```diff
@@ -0,0 +1,173 @@
+"""
+This Pytest  script tests stac_parent_update.py, notify_to_stac_queue.py as well as
+the severless lambda function given in stac.py
+"""
+
+# pylint: disable=W0621, W0613
```

Comment:
Whenever disabling pylint warnings, use the extended name `unused-argument` and `redefined-outer-name`, and mention *why* they're disabled. In this case, it's to for using pytest fixtures.